src/data/contributors/harshitkumarojha.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Harshit Ojha",
+    "skills" : ["html", "css", "bootstrap", "discord js", "React"],
+    "cause" : "Help making world become a better place to live :D",
+    "image" : "Avatar3",
+    "github" : "https://github.com/HarshitKumarOjha",
+    "discod" : "https://discord.gg/59HSBfT",
+    "quora" : "https://www.quora.com/profile/Harshit-Kumar-Ojha"
```

Comment:
unfortunately, Discord and quora links can't be rendered on the web. Only github, link will be rendered.

Reply:
removed the discord and quora links 🤗

Reply:
sorry for disturbing but i just want to say that im a beginner and i did some mistakes...but now i fixed the code,
hope u'll merge the pr thank you :)